modules/core/ce2/src/main/scala/tofu/interop/CE2Agents.scala

```diff
@@ -38,6 +38,12 @@ final case class SerialSemRef[F[_]: Monad, A](ref: Ref[F, A], sem: Semaphore[F])
     modifyM(a => if (f.isDefinedAt(a)) f(a) else (default, a).pure[F])
 }
 
+/** Default implementation of [[tofu.concurrent.Permit]] that use [[cats.effect.concurrent.Semaphore]]
+  */
+final case class PermitSem[F[_]](sem: Semaphore[F]) extends Permit[F] {
```

Comment:
I suggest using the full name, we don't gain anything from typing Sem instead of Semaphore

Reply:
Hi, thx, I renamed it to PermitSemaphore